plugins/chat/assets/javascripts/discourse/components/chat-channel-card.gjs

```diff
@@ -36,76 +36,59 @@ export default class ChatChannelCard extends Component {
             {{#if @channel.chatable.read_restricted}}
               {{dIcon "lock" class="chat-channel-card__read-restricted"}}
             {{/if}}
-          </LinkTo>
-
-          <div class="chat-channel-card__header-actions">
             {{#if @channel.currentUserMembership.muted}}
-              <LinkTo
-                @route="chat.channel.info.settings"
-                @models={{@channel.routeModels}}
-                class="chat-channel-card__tag -muted"
-                tabindex="-1"
-              >
-                {{i18n "chat.muted"}}
-              </LinkTo>
+              <span
+                class="chat-channel-card__muted"
+                aria-label="Channel is muted."
```

Comment:
we an i18n locale here

Reply:
Also do we need aria-label given we have a title`?

Reply:
oh right forgot thx

Reply:
> Also do we need aria-label given we have a title`?

Yes, aria-label for screenreaders, title for hover tooltip.